### ISSUE TITLE
Last activity - this is time since last xAPI activity in days."

### ISSUE THREAD

**Issue by michaelwebbjisc** [jiscdev/ssp-lite#1]
@amason asked:

@michaelwebbjisc I need to check these requirements:

"Last Activity
Within a week, indicator should appear green in UI
Within 2 weeks, indicator should appear amber in UI
Over 2 weeks, indicator should appear red in UI

Calculation Requirements
Last activity - this is time since last xAPI activity in days."

Is that still accurate because Paul Spaude advised he thought the requirements changed to gray for scenarios. I have no record of that requirement change so can you advise if you wish to see the color based on the specifics above OR do you just want that field gray regardless of the last activity time?

**Comment by michaelwebbjisc** [comment]
Yes, colour based on the above.  I guess Paul may be right and I might have changed my mind! I can't remember!

**Comment by amason** [comment]
@michaelwebbjisc  I put this in another thread but it belongs here:

We see the issue now and it's on our end with Last Activity Time and Last Graded Work Date.
Here are the requirements you gave us:

"3. Last Activity
Within a week, indicator should appear green in UI
Within 2 weeks, indicator should appear amber in UI
Over 2 weeks, indicator should appear red in UI

Date of Last Graded Work
Only text should display reflecting the Date of Last Graded Work. It should still behave as an alert, but do not color code this risk indicator in the UI
The value of the Date of Last Graded Work will be changing often based on when the faculty last graded work."
In SSP, we coded it backwards. We coded Last Activity Time to be a Date and Time stamp and Last Graded Work Date to be a number. We need to flip that in SSP so that Last Activity Time is in days and Last Graded Work Date is Date/time stamp.

So - let us get this code fixed and I'll then send you a new test scenario. Thanks and sorry for the confusion on this!
